Apps/Examples/ExamplesUITests/ExamplesUITests.swift

```diff
@@ -32,13 +32,22 @@ final class ExamplesUITests: XCTestCase {
         })
 
         // Navigate to an example that should trigger location permissoon alert to be shown
-        let searchField = app.navigationBars.firstMatch.searchFields.firstMatch
+        let searchField = app.searchFields.firstMatch
         searchField.tap()
         searchField.typeText("Location")
         app.tables.firstMatch.cells.firstMatch.tap()
 
-        // wait for the alert to appear
-        sleep(2)
+        // pre iOS 13 solution
+        let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
+
+        let allowBtn = springboard.buttons["Allow"]
+
+        // Wait for iOS 12 popup. On iOS 13+ we also need that
+        // to leave some room for monitor to catch alert
+        if allowBtn.waitForExistence(timeout: 2) {
```

Comment:
I'd rather have this with explicit branching, so that when iOS 12 gets dropped it'd be easier to remove no longer needed code.
```swift
if #available(iOS 13, *) {
   sleep(2)
} else {
   ...
}
```

Reply:
Makes sense!